src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -57,7 +57,9 @@ class Referral(
   var concludedAt: OffsetDateTime? = null,
 
   // draft referral fields
-  @OneToOne(mappedBy = "draftReferral", cascade = [CascadeType.ALL]) @PrimaryKeyJoinColumn var serviceUserData: ServiceUserData? = null,
+  @OneToOne(mappedBy = "referral", cascade = [CascadeType.ALL])
```

Comment:
May not be an issue at all, but noticed the mapping has changed from draftReferral to referral?
Just checking this was intentional

Reply:
Yes this is intentional Ryan (I've looked back over the blame logs), the new version of hibernate won't let you use the same mapping class to reference two different entities, so draft referral will lead to error.